test/test_configuration.rb

```diff
@@ -335,6 +335,18 @@ class TestConfiguration < JekyllUnitTest
         Jekyll.configuration(test_config.merge({ "config" => @paths[:other] }))
     end
 
+    should "load different config if specified with symbol key" do
+      allow(SafeYAML).to receive(:load_file).with(@paths[:default]).and_return({})
```

Comment:
(in the case when the config option is not recognited properly, this line produces a regular test failure, instead of an exception due to wrongly set up stub)